doc/user/content/releases/v0.27.0.md

```diff
@@ -1,47 +1,127 @@
 
-* Require associating indexes with a cluster.
+* Add [materialized views](/sql/create-materialized-view), which are views that
+  are persisted in durable storage and incrementally updated as new data
+  arrives.
 
-* Remove the `MATERIALIZED` keyword as shorthand for `CREATE DEFAULT INDEX`.
+  The result of a materialized view is not maintained in memory, unless you
+  create an [index](/sql/create-index) on it. (Intermediate state necessary for
+  efficient incremental updates of the materialized view may be maintained in
+  memory, however.)
 
-  * Remove `CREATE MATERIALIZED SOURCE` syntax.
-  * Remove the `materialized` column from the output of `SHOW FULL SOURCES`.
+* Add [connections](/sql/create-connection/), which describe how to connect to
+  and authenticate with external systems. Once created, a connection is reusable
+  across multiple [`CREATE SOURCE`](/sql/create-source) and
+  [`CREATE SINK`](/sql/create-sink) statements.
 
-* Add recorded views (which will be conflated with materialized views in an
-  upcoming release {{% gh 13663 %}}).
+* Add [secrets](/sql/create-secret), which securely store sensitive credentials
+  (like passwords and SSL keys) for reference in connections.
 
-* Add secrets.
+* Durably record data ingested from [sources](/sql/create-source).
 
-* Add connections.
+  Once a source has acknowledged data upstream (e.g., via committing a Kafka
+  offset or advancing a PostgreSQL replication slot), it will never re-read that
+  data. As a result, PostgreSQL sources no longer have a "single
+  materialization" limitation. All sources are directly queryable via
+  [`SELECT`](/sql/select).
 
-* Ingest and persist sources. This removes the single-materialization limitation
-  for sources. Sources can always be queried directly.
+* Allow provisioning the size of a source or sink.
 
-* Require the use of connections with sources and sinks.
+  Each source and sink now runs with an isolated set of compute resources. You
+  can adjust the size of the resource allocation with the
+  [`SIZE`](/sql/create-source/#sizing-a-source) parameter.
 
-* Allow provisioning the size of a source or sink.
+* Add [load generator sources](/sql/create-source/load-generator), which
+  produce synthetic data for use in demos and performance tests.
+
+* Add an [HTTP API](/integrations/http-api) which supports executing SQL queries
+  over HTTP.
+
+* **Breaking change.** Require all [indexes](/sql/create-index) to be associated
+  with a cluster.
+
+* **Breaking change.** Require the use of [connections](/sql/create-connection/)
+  with Kafka sources, PostgreSQL sources, and Kafka sinks.
+
+* **Breaking change.** Rename `TAIL` to [`SUBSCRIBE`](/sql/subscribe).
+
+* **Breaking change.** Change the meaning of `CREATE MATERIALIZED VIEW`.
+
+  `CREATE MATERIALIZED VIEW` now creates a new type of object called a
+  [materialized view](/sql/create-materialized-view), rather than creating a
+  view with a default index.
```

Comment:
```suggestion
  `CREATE MATERIALIZED VIEW` now creates a new type of object called a
  [materialized view](/sql/create-materialized-view), rather than providing 
  a shorthand for creating a view with a default index.
```

Reply:
Applied locally.